### DISCUSSION TITLE
SSIM loss 3D, 1 channel

### DISCUSSION THREAD

**Discussion by ramdhan1989** [Project-MONAI/tutorials#1622] [General]
Hi,
Can I use SSIM loss for 3D with 1 channel data?

thanks

**Comment by KumoLiu** [comment]
Sure, yes.
https://github.com/Project-MONAI/MONAI/blob/abe2e3151e77fdbcad68268111aaabf1e73c3dbe/monai/losses/ssim_loss.py#L116-L124
